cirq/ops/common_gates_test.py

```diff
@@ -610,6 +680,44 @@ def test_measurement_channel():
                        [0, 0, 0, 0],
                        [0, 0, 0, 0],
                        [0, 0, 0, 1]])))
+    np.testing.assert_allclose(
+            cirq.channel(cirq.MeasurementGate(2, qid_shape=(2, 3))),
+            (np.array([[1, 0, 0, 0, 0, 0],
```

Comment:
`np.diag(cirq.one_hot` would shorten this up.

Reply:
Shortened.